src/main/app/claims/responseModelConverter.ts

```diff
@@ -233,6 +267,17 @@ export class ResponseModelConverter {
     }
   }
 
+  private static convertPartPaymentDate (partAdmission: PartialAdmission): Moment {
```

Comment:
We have exactly same method in `responseModelConverter.ts`, please change the parameter and reuse same